SherpaFrontEnd/Pages/TeamContent/TeamContent.razor

```diff
@@ -20,14 +20,16 @@
 
     <TabControl InitialTab="@Tab">
         <TabPage Text="Analysis">
-            <button>Send a new survey</button>
+            <button @onclick="@(RedirectToLaunchSurveyPage)" type="button" class="bg-primary-red-principal-800 hover:bg-primary-red-900 rounded-lg px-6 py-2 text-white font-semibold text-sm">
```

Comment:
you don't need to do "@()" inside the @onclick
@onclick="RedirectToLaunchSurveyPage" should work